### ISSUE TITLE
403 error after update today

### ISSUE THREAD

**Issue by Draculetta72** [yuliskov/SmartTube#3626]
Did update. Now nothing will load. Every video says unknown error response code 403

**Comment by cybrnook** [comment]
Same. Was fine before today's update.

**Comment by bigshiny90** [comment]
Same. Latest update completely killed ability to play anything.

was just fine before latest.  Please fix soon? Revert?

**Comment by Daniel-stock** [comment]
Habrá alguna solución ?

**Comment by ACR-Jeff** [comment]
Duplicate https://github.com/yuliskov/SmartTube/issues/3606

**Comment by bobnecat** [comment]
It's a duplicate issue as it's not caused by the latest update.  No need to create second thread and instead post in the original one.

**Comment by bigshiny90** [comment]
For many, it WAS caused by today’s update, so that’s important to note I would think.
Yesterday all worked fine. (Android TV)

> It's a duplicate issue as it's not caused by the latest update. No need to create second thread and instead post in the original one.

**Comment by EdmundGerber** [comment]
I didn't update - still on version 22.35 - and now getting 403 errors. It's nothing to do with the 22.37 update - it's something google are rolling out on their end.

**Comment by baby-fox88** [comment]
Same here

**Comment by bigshiny90** [comment]
Yup, making sense.. I’m still ready thru the main thread for this issue and starting to see

**Comment by macmarine20** [comment]
Multiple live streams are working for me, but I can't get a single video to play without getting the 403 error.

**Comment by ErvinMark** [comment]
Not working here also 403 always even I roll to older version same thing !

**Comment by acostaja** [comment]
Same issue err 403 just after today update 🫣

**Comment by brunobelo** [comment]
Dudes, it wasn't the update. I had the error without the update, made the update and then the error continued. It was not the update!

**Comment by cybrnook** [comment]
Why so hostile? I think there are two different groups of people here, those who had the issue before today's update and then those (like myself) who only had the issue start immediately after today's update. Spamming is just going to cause confusion in every report. No need to be hostile.

**Comment by faroukbmiled** [comment]
> For many, it WAS caused by today’s update, so that’s important to note I would think. Yesterday all worked fine. (Android TV)
> 
> > It's a duplicate issue as it's not caused by the latest update. No need to create second thread and instead post in the original one.

it's not caused by the latest update bruh, stop spreading misinformation, I'm on 22.35 and getting 403 all of a sudden ,youtube is definitely doing something

**Comment by TroglodyteGuy** [comment]
> > For many, it WAS caused by today’s update, so that’s important to note I would think. Yesterday all worked fine. (Android TV)
> > > It's a duplicate issue as it's not caused by the latest update. No need to create second thread and instead post in the original one.
> 
> it's not caused by the latest update bruh, stop spreading misinformation, I'm on 22.35 and getting 403 all of a sudden ,youtube is definitely doing something

I was thinking the same with YouTube doing something. I am wondering if it's the YouTube ad injection process breaking SmartTube?

**Comment by baby-fox88** [comment]
But live videos still work? 👀🤔

**Comment by Vickydjay3232** [comment]
Facing the same

**Comment by Draculetta72** [comment]
Update fixed this for now. Until YT pushes new ad injection

**Comment by Daniel-stock** [comment]
Eh solucionado el error 403, lo que e hecho fue inhabilitar YouTube y actualizar a la versión que me salía a mi, espero les ayude

**Comment by brunobelo** [comment]
> Why so hostile? I think there are two different groups of people here, those who had the issue before today's update and then those (like myself) who only had the issue start immediately after today's update. Spamming is just going to cause confusion in every report. No need to be hostile.

Oh, I wasn't trying to be hostile, I'm sorry if it looked like that.

**Comment by brunobelo** [comment]
> Why so hostile? I think there are two different groups of people here, those who had the issue before today's update and then those (like myself) who only had the issue start immediately after today's update. Spamming is just going to cause confusion in every report. No need to be hostile.

Oh, I wasn't trying to be hostile, I'm sorry if it looked like that.

**Comment by Tenderfoot56** [comment]
I just wanted to add that I too am having this problem with the latest update.  I disabled VPN, cleared the cache and data, uninstalled, rebooted and reinstalled, etc.  Live streaming channels just fine but normal youtube videos come back with this error as shown at the top of this post.  I'm running the app on a 2nd gen FireTV Cube and also an older FireTV/Sound bar combo device from about 2 years ago installed on a different TV.  Both exhibit the same problem.  I love the app and use it frequently, so I was sad to see this source error 403 pop up a couple of days ago.  Actually, on the Sound bar, the error popped up with the prior version of the app.  Updating it did not resolve the issue on either device as of late in the day on July 9, 2024. Thought I would share this with everyone to let you know this is not an isolated incident.
